src/app/utilities/http-methods/request.js

```diff
@@ -102,7 +95,7 @@ export default function request(method, URI, willRetry = true, onRetry = () => {
                 responseIsText = response.headers.get('content-type').match(/text\/plain/);
             } catch (error) {
                 console.error(`Error trying to parse content-type header`, error);
-                log.add(eventTypes.unexpectedRuntimeError, {message: `Error trying to parse content-type header: ${JSON.stringify(error)}`});
+                log.event("Error trying to parse content-type header", log.error(error))
```

Comment:
is the content-type header included in `error`? if not, might be worth adding it to the `data` attribute